frontend/pages/hosts/details/cards/Software/HostSoftwareTableConfig.tsx

```diff
@@ -119,6 +119,7 @@ export const generateSoftwareTableHeaders = ({
           formatter={() =>
             formatSoftwareType({
               source: cellProps.cell.value as SoftwareSource,
+              extension_for: cellProps.row.original.extension_for,
```

Comment:
Same point as [here](https://github.com/fleetdm/fleet/pull/33595/files#r2408362057), we should just be calculating the formatted value at this scope and plugging it in to the `value` prop